src/content/docs/iast/introduction.mdx

```diff
@@ -35,9 +35,8 @@ IAST helps you:
   Go to <DNT>**[one.newrelic.com](https://one.newrelic.com) > All capabilities > IAST**</DNT> to open the IAST testing status page.
 </figcaption>
 
-IAST is fully integrated with [New Relic Vulnerability Management](/docs/vulnerability-management/overview/), allowing you to continuously and quickly find, fix, and verify high-risk vulnerabilities across the software development lifecycle. IAST's exclusive proof of exploits helps you use your time more efficiently by providing you with a high-priority vulnerability, the steps to reproduce it, and remediation guidante.
-
-Our <InlinePopover type="apm"/> agents deliver New Relic IAST and you can enable it with a simple configuration setting change. IAST is available in these languages:
+IAST is a part of the New Relic APM agent, and only requires a security configuration to be enabled. It can be fully integrated with [New Relic Vulnerability Management](/docs/vulnerability-management/overview/), allowing you to continuously and quickly find, fix, and verify high-risk vulnerabilities across the software development lifecycle.
```

Comment:
The problem with "continuously" and "quickly" is that they're not, technically, very specific.

I'd recommend replacing with "allowing you to find, fix, and verify high-risk"